<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导入图像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导出图像</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>800</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="outputModeBox">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>130</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="outputLabel">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>130</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>输出模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>170</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>高斯核尺寸</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>210</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>边缘简化阈值</string>
   </property>
  </widget>
  <widget class="QSlider" name="blursizeSlider">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>170</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="pageStep">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>250</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>迭代次数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>290</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>显著性阈值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>390</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>着色风格</string>
   </property>
  </widget>
  <widget class="QComboBox" name="colorizeModeBox">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>390</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>330</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>采样点增量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>610</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="epsilonSpinBox">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>210</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="itrateSpinBox">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>250</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="saliencyThereSpinBox">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>290</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>64</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>760</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>610</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>重设为默认值</string>
   </property>
  </widget>
  <widget class="QSlider" name="sampleNumSlider">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>330</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveModelButton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导出模型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>430</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>深度上限</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="depSpinBox">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>430</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="blurValueLabel">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>170</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sampleValueLabel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>360</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="recToolButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>560</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>矩形选框</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="lassoToolButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>520</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>套索</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dragToolButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>480</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>拖拽移动</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
